dd-trace-ot/src/main/java/datadog/opentracing/DDTracer.java

```diff
@@ -217,6 +219,28 @@ public DDTracer(
       final Map<String, String> serviceNameMappings,
       final Map<String, String> taggedHeaders,
       final int partialFlushMinSpans) {
+    this(
+        serviceName,
+        writer,
+        sampler,
+        localRootSpanTags,
+        defaultSpanTags,
+        serviceNameMappings,
+        taggedHeaders,
+        partialFlushMinSpans,
+        Config.DEFAULT_MAX_SPANS_PER_TRACE);
```

Comment:
Where is `Config.get().getMaxSpansPerTrace()` being used?

Reply:
Nice catch, updating to use configured value in all cases.